step.yml

```diff
@@ -41,6 +41,17 @@ inputs:
       description: ""
       is_required: true
 
+  - lint_range:
+    opts:
+      category: Config
+      title: "Lint changed files only"
+      summary: ""
+      description: ""
```

Comment:
Could you add a little description telling oh it does? 
Like "Take the git diff and only lint those files"

Reply:
Okay I'll add description and change title then commit